test/ui-testing/pick-lists.js

```diff
@@ -0,0 +1,268 @@
+      });
+
+      it('should open Settings', done => {
+        helpers.clickSettings(nightmare, done);
+      });
+
+      it(`should create pick list ${pickList} in settings`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-lists"]')
+          .click('a[href="/settings/licenses/pick-lists"]')
+          .waitUntilNetworkIdle(1000)
+          .wait('#clickable-add-pick-lists')
+          .click('#clickable-add-pick-lists')
+          .wait('input[name="items[0].desc"]')
+          .type('input[name="items[0].desc"]', pickList)
+          .wait('#clickable-save-pick-lists-0')
+          .click('#clickable-save-pick-lists-0')
+          .wait(222)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should find pick list ${pickList}`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-lists"]')
+          .click('a[href="/settings/licenses/pick-lists"]')
+          .wait('#editList-pick-lists')
+          .evaluate(list => {
+            const listRows = [...document.querySelectorAll('#editList-pick-lists')].map(e => e.textContent);
+            const listElement = listRows.find(r => r.indexOf(list) >= 0);
+            if (!listElement) {
+              throw Error(`Could not find row with the list ${list} but found ${listElement}`);
+            }
+          }, pickList)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should edit pick list ${pickList} in settings`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-lists"]')
+          .click('a[href="/settings/licenses/pick-lists"]')
+          .waitUntilNetworkIdle(1000)
+          .wait('#clickable-edit-pick-lists-0')
+          .click('#clickable-edit-pick-lists-0')
+          .wait('input[name="items[0].desc"]')
+          .insert('input[name="items[0].desc"]', '')
+          .insert('input[name="items[0].desc"]', editedPickList)
+          //  .type('input[name="items[0].desc"]', editText)
+          .wait('#clickable-save-pick-lists-0')
+          .click('#clickable-save-pick-lists-0')
+          .wait(222)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should find edited pick list ${editedPickList}`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-lists"]')
+          .click('a[href="/settings/licenses/pick-lists"]')
+          .wait('#editList-pick-lists')
+          .evaluate(list => {
+            const listRows = [...document.querySelectorAll('#editList-pick-lists')].map(e => e.textContent);
```

Comment:
I don't think this `evaluate` is working as intended. The `querySelectorAll` is just going to return the single element for the item by the ID `editList-pick-lists`.

You could try `[...document.querySelectorAll('#editList-pick-lists [role="row"]')]` to fetch the _rows_ rather than the list itself.